README.md

```diff
@@ -49,6 +49,29 @@ Example Playbook
 Generate var file mapping [demo](https://youtu.be/s1-Hq_Mk1w8)<br/>
 Fail over scenario [demo](https://youtu.be/mEOgH-Tk09c)
 
+Scripts
+-------
+The ovirt-dr script should provide the user a more convinient way to run
```

Comment:
`ovirt-dr` where is it installed in /usr/bin?

Reply:
It should be somewhere there..

Reply:
I'm not sure how to do that.
I prefer to do that on another patch so at least we will have this.
Meanwhile I've added a commit to this PR of spec file

Reply:
Ok, np, just that user must enter full path, which is different in galaxy and in RPM installation.

Reply:
Should I add this in the spec as well?